sky/backends/cloud_vm_ray_backend.py

```diff
@@ -4402,7 +4403,8 @@ def post_teardown_cleanup(self,
                               handle: CloudVmRayResourceHandle,
                               terminate: bool,
                               purge: bool = False,
-                              remove_from_db: bool = True) -> None:
+                              remove_from_db: bool = True,
+                              failover: bool = False) -> None:
```

Comment:
Can we add some note in the docstring about what `failover` does? (Skip some cleanup that is global.)

Reply:
Updated